utils/_context/_scenarios/__init__.py

```diff
@@ -105,6 +105,26 @@ def all_endtoend_scenarios(test_object):
         doc="Test W3C trace style",
     )
 
+    trace_propagation_style_w3c_datadog_b3 = EndToEndScenario(
+        "TRACE_PROPAGATION_STYLE_W3C_DATADOG_B3",
```

Comment:
Side question, would it be possible to integrate this conf in an existing scenario, for instance `tracing_config_nondefault_2` and `tracing_config_nondefault_3`.

There are several advantages to use existing scenarios : 

* slower CI time
* higher probability to catch interaction issues
* less work to do to integrate the new scenario on existing CI 

Actually, if it's possible, we should not create individual scenario unless no other scenario is compatible with the conf needed for the new test.